torchao/kernel/README.md

```diff
@@ -0,0 +1,19 @@
+## Autotuner and custom Triton kernels
+
+### Use case
+
+### How to contribute
+
+### Environment variables
+
+`TORCHAO_AUTOTUNER_ENABLE=0`
+
+Set this to a nonzero value to enable the kernels generated by the autotuner. This is turned off by default, because it is still an experimental feature and also can take a long time to run.
+
+Searching a new config can take a long time and we'll save the updated data in `data.pkl`. If you'd like to contributed updated configs for your hardware or shapes, please open a pull request.
```

Comment:
presumably people won't contribute the pickle file since that's not human readable? Also kind of a security issues for us to host pickle files

Reply:
I added https://github.com/pytorch-labs/ao/pull/41/files#diff-4986e1d3257adc0a73b17fd6f21ef9d3b2c0eaec9027381e6f2de89e5be0e6b5 to make it easier to inspect. It stores the triton Configs, so it's a bit more difficult to make them human readable by default.